### ISSUE TITLE
Theme no longer works in BBEdit 11 and TextWrangler 5

### ISSUE THREAD

**Issue by Honestpuck** [rcarmo/textwrangler-bbedit-solarized#9]
It appears that the entire colour scheme system has changed in the new versions of BBEdit and TextWrangler and these schemes no longer work at all.

**Comment by ozymandias1** [comment]
Try this:

https://github.com/ozymandias1/Solarized-Dark-textWrangler5-bbEdit10-

**Comment by Honestpuck** [comment]
Yep, that works for "Dark" - now one for "Light" would be my answer. :)

**Comment by venkatasg** [comment]
It doesn't work for bbedit 12, and neither does the new scheme by @ozym11
